Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -896,6 +931,10 @@ def interval_in_sec(val):
 
 
 def test_module(client, args):
+    if client.collection not in client.all_collections:
```

Comment:
Good job on adding this check, but as it is I still feel that `client.all_collections` can be a function instead of a member, as it'll be called only once every time the code runs, and will either exit right after or ignore its value.

Keep in mind this call alone can take a few seconds on a slow TAXII server (such as hailataxii).

Reply:
You are right